.gitignore

```diff
@@ -1,3 +1,8 @@
+# generated parser / lexer
+/lib/rubocop/ast/node_pattern/parser.racc.rb
+/lib/rubocop/ast/node_pattern/parser.output
+/lib/rubocop/ast/node_pattern/lexer.rex.rb
```

Comment:
So it is ignored but included in the gem distribution.
Will it make it impossible to `gem 'rubocop-ast', github: 'rubocop-hq/rubocop-ast'`?

Reply:
I used the same setup as `parser` gem.
Dunno, probably not...